integration-tests/kyuubi-jdbc-it/src/test/scala/org/apache/kyuubi/it/jdbc/impala/WithKyuubiServerAndImpalaContainer.scala

```diff
@@ -0,0 +1,59 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kyuubi.it.jdbc.impala
+
+import java.nio.file.{Files, Path, Paths}
+
+import org.apache.kyuubi.{Utils, WithKyuubiServer}
+import org.apache.kyuubi.config.KyuubiConf
+import org.apache.kyuubi.config.KyuubiConf.{ENGINE_JDBC_EXTRA_CLASSPATH, KYUUBI_ENGINE_ENV_PREFIX, KYUUBI_HOME}
+import org.apache.kyuubi.engine.jdbc.impala.WithImpalaEngine
+
+trait WithKyuubiServerAndImpalaContainer extends WithKyuubiServer with WithImpalaEngine {
+
+  private val kyuubiHome: String = Utils
+    .getCodeSourceLocation(getClass).split("integration-tests").head
+
+  private val hiveJdbcConnectorPath: String = {
+    val keyword = "kyuubi-hive-jdbc-shaded"
```

Comment:
In CI, we explicitly set `-Pjdbc-shaded` to use the `kyuubi-hive-jdbc-shaded` for testing, let's make the test also works without `-Pjdbc-shaded`(`kyuubi-hive-jdbc` is used in such a case)

```suggestion
    val keyword = "kyuubi-hive-jdbc"
```

Reply:
At first I used `${hive.jdbc.artifact}` in integration tests, but in case of non-shaded driver artifact, test failed with `ClassNotFoundException` of driver dependencies (for instance, `org.apache.http.client.HttpClient`). So, in my opinion, here we should use self-contained driver fat jar with all needed dependencies, which is what a shaded artifact is.

Reply:
Also, the test will pass without `-Pjdbc-shaded` if the updated Kyuubi JDBC driver with fixed `getMoreResults()` method will be available in snapshot maven repo, so I'll create separate PR with that fix as you've suggested in https://github.com/apache/kyuubi/pull/6104#discussion_r1505826643